llama_index/llms/openai.py

```diff
@@ -181,12 +186,9 @@ def _get_all_kwargs(self, **kwargs: Any) -> Dict[str, Any]:
         return {**self._credential_kwargs, **self._model_kwargs, **kwargs}
 
     def _chat(self, messages: Sequence[ChatMessage], **kwargs: Any) -> ChatResponse:
-        if not self._is_chat_model:
-            raise ValueError("This model is not a chat model.")
-
         message_dicts = to_openai_message_dicts(messages)
         response = completion_with_retry(
-            is_chat_model=self._is_chat_model,
```

Comment:
Since it was already checked for upstream in the caller of this private method, it's also unnecessary to re-compute this here